main.py

```diff
@@ -87,6 +88,14 @@ def stored_flows_list(self, dpid):
         """Ordered list of all stored flows given a dpid."""
         return itertools.chain(*list(self.stored_flows[dpid].values()))
 
+    def stored_flows_by_state(self, dpid, state):
+        """Get stored flows dict filter by a state."""
+        filtered_flows = {}
+        for entry in self.stored_flows_list(dpid):
+            if entry.get("state") and entry["state"] == state:
+                filtered_flows[entry["_id"]] = entry
+        return filtered_flows
```

Comment:
```suggestion
        for entry in self.stored_flows_list(dpid):
            if entry.get("state") == state:
                filtered_flows[entry["_id"]] = entry
        return filtered_flows
```

Reply:
I've just commit the suggestion, thank you.